mkdocs.yml

```diff
@@ -8,17 +8,19 @@ edit_uri: edit/dev/docs/
 nav:
   - Getting started:
     - Introducing episb: README.md
-  - How-to guides:
+  - HUB user guides:
     - Query types explained: howto-query-provider.md
     - Add a new provider to a hub: howto-subscribe.md
     - Normalize to a segmentation: howto-normalize.md
+  - HUB/Provider administrator guides:
     - Build a data provider: howto-build-provider.md
     - Build a hub: howto-build-hub.md
+  - HUB/Provider programmer's guides:
+    - Hub API: hub-api.md
```

Comment:
API documentation should clearly go under reference, I think...

A howto would be a bit more...friendly.

ideally the API documentation can be auto-generated from code comments.

Reply:
Got it. Yes, we will auto-generate API docs - scalatra has that capability, I just need to turn it on and make it happen.